content/docs/engine-dev/tools/project-manager/_index.md

```diff
@@ -1,167 +1,16 @@
 ---
 linkTitle: Project Manager
-title: Project Manager Developer High Level Overview (Boot-up sequence)
-description: High Level Developer Overview for the Project Manager tool.
+title: Project Manager Developer Documentation
+description: Documentation for developers extending or contributing to the Project Manager bundled as part of Open 3D Engine.
+weight: 400
```

Comment:
Leave out the weight here. For _index.md files in a subdirectory of the tools directory, this just affects the order of sections underneath "Tools". By removing any weights, it will just be alphabetical by tool name.